content/applications/hr/payroll/contracts.rst

```diff
@@ -42,19 +39,19 @@ Required fields
    :align: center
    :alt: New contract form to be filled in when creating a new contract.
 
```

Comment:
I totally understand that you prefer to break down forms like this into 'Require' and 'Optional' fields...however, I think it would be best if you simply list the fields in order as they appear on the form, then add in the explanation if they are **required**. That way, the reader doesn't feel like they have to jump around, and scroll back and forth between sections, as they familiarize themselves with the form and its fields. So, if you could implement that adjustment, I think it would greatly benefit the overall doc. Thanks! 👍